Client/Frontend/Browser/TabTrayController.swift

```diff
@@ -651,7 +651,9 @@ extension TabTrayController: UIScrollViewAccessibilityDelegate {
         visibleCells = visibleCells.filter { !CGRectIsEmpty(CGRectIntersection($0.frame, bounds)) }
 
         let cells = visibleCells.map { self.collectionView.indexPathForCell($0)! }
-        let indexPaths = cells.sort { $0.section < $1.section || ($0.section == $1.section && $0.row < $1.row) }
+        let indexPaths = cells.sort { (a: NSIndexPath, b: NSIndexPath) -> Bool in
```

Comment:
Nit: The compiler can't infer that a, b are `NSIndexPaths`? Would be a bit cleaner to drop the types.